src/components/ListItem.tsx

```diff
@@ -1,18 +1,101 @@
+import MoreHorizIcon from "@mui/icons-material/MoreHoriz";
+import { Button, IconButton } from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import AddCardIcon from "@mui/icons-material/AddCard";
+import { useState } from "react";
+import CloseIcon from "@mui/icons-material/Close";
+import FormControlUnstyled from "@mui/base/FormControlUnstyled";
+import { TextField } from "@mui/material";
+import CardItem from "./carditem";
 
-interface  props {
-    title: string,
-}
+// interface props {
+//   title: string;
+// }
 
-const CardProps = {
-    title: 'rfergv',
+interface cardbox {
+  listcard: {list: string}[];
 }
 
-export default function ListItem(props: props) {
-    return (
-        <div
-         >
-            <div>{CardProps.title}</div>
-            <div>ewfderf</div>
-         </div>
-    );
+
+export default function ListItem(props: { title: string }) {
+  const [handlecard, setHandleCard] = useState(false);
+  const cardHandler = () => {
+    setHandleCard(true);
+  }
+  const [title, setTitle] = useState("");
+  const [titlelist, setTitleList] = useState<cardbox[]>([]);
+
+  const buttonClick = () => {
+    if(title.trim() !== '') {
```

Comment:
use title.trim.length

Reply:
trim.length is no working... not getting output with using title.trim.length !== 0